prow/github/types.go

```diff
@@ -210,3 +211,38 @@ type Commit struct {
 	Removed  []string `json:"removed"`
 	Modified []string `json:"modified"`
 }
+
+// ReviewEvent is what GitHub sends us when a PR review is changed.
+type ReviewEvent struct {
+	Action      string `json:"action"`
+	PullRequest `json:"pull_request"`
```

Comment:
To expand: embedding the struct like this makes the parent struct take on the methods of child structs. That's not a problem here, but can potentially cause bugs down the line.